aws/resource_aws_emr_managed_scaling_policy_test.go

```diff
@@ -0,0 +1 @@
+package aws
```

Comment:
For more information about implementing the required testing, please see the [Contributing Guide section on Writing Acceptance Tests](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md#resource-acceptance-testing).